packages/fxa-settings/package.json

```diff
@@ -26,14 +26,14 @@
       "src/**/*.{ts,tsx,js}",
       "!src/**/*.stories.*",
       "!src/**/mocks.*",
-      "!src/components/*(ScrollToTop|UnitRowRecoveryKey|PageAvatar|App)/*"
+      "!src/Settings/components/*(ScrollToTop|UnitRowRecoveryKey|PageAvatar|App)/*"
```

Comment:
A couple notes/questions about the `Settings` directory:
* Starting with capitalized letter in a React app conventionally means that there is going to be a React component directly inside of that directory. The way it's used here if we keep this path and without its own component, I think a lowercase `settings` makes more sense 🫤
* I personally think the path `components/settings/MySettingsComponent` might be nicer. [This file tree](https://github.com/mozilla/fxa/tree/d8eee8c5fc85c066ca6293074e956d9eba85339c/packages/fxa-settings/src) looks odd to me since there are two `components` directories. (For this kinda thing, I'll usually hit the team up to see what the general consensus is.) <img width="276" alt="image" src="https://user-images.githubusercontent.com/13018240/197865997-1c850eea-05d4-4cdb-9667-f13bd47e6c9c.png">
* Some files didn't seem to retain their git histories, like `packages/fxa-settings/src/Settings/components/App/en-US.ftl` for example. Can that be mitigated a little better or did you already try moving all of these with `git mv`?

Reply:
Will do, these are great suggestions!